docs/manual/getting_started.rst

```diff
@@ -347,41 +469,36 @@ macOS in the install or build directory:
 
     rm -rf *
 
-Uninstallation for the product installer
+Uninstall Cyclone DDS product installer
 ''''''''''''''''''''''''''''''''''''''''
-
 Windows
 ^^^^^^^
 
-On Windows, to uninstall the Cyclone DDS you can either do it from the
+On Windows, to uninstall Cyclone DDS, you can either do it from the
 Windows application control panel (Programs and Features in Control
 Panel) or by using the product installer; in this case, start-up the
-Cyclone DDS product installer, and select 'Remove'.
+Cyclone DDS product installer, and select 'Remove.'
```

Comment:
This should also be defined clearly in a style guide. Because I would expect from this that the `Remove` button would have a period as well, instead of the period ending the sentence. I'm aware that some people might include the terminating punctuation within the quote, but it leads to more confusion here (which we want to avoid).